index.html

```diff
@@ -1382,7 +1382,7 @@ <h3>Marie Bellevue
                         <figcaption class="caption text-center">
                             <h3>Aniruddh Khanwale
                                 <!-- <small>loves making google forms</small> -->
-                                <small class="major">Materials Science & Engineering, EECS</small>
+                                <small class="major">Materials Science and Engineering, EECS</small>
```

Comment:
There's nothing wrong here, but do you mind adding Energy Engineering at the end :O